src/TypeSpec.Extension/Emitter.Csharp/src/lib/confidentLevels.ts

```diff
@@ -0,0 +1,78 @@
+import {
```

Comment:
Please add the header for this new file.

Reply:
Fixed but we should have a CI for that.